src/api/nft.ts

```diff
@@ -106,4 +107,7 @@ export interface Nft extends BaseNft {
 
   /** Detailed information on why an NFT was classified as spam. */
   spamInfo?: SpamInfo;
+
+  /** Time at which the NFT was most recently acquired by the user. */
+  acquiredAt?: AcquiredAt;
```

Comment:
cool to use this type in both raw and user-facing response?

Reply:
let's keep them separate. it's in a little messy in v2, but raw and user-facing interfaces are all separate in v3

Reply:
Add a comment that this is only available when specifying the orderby = transfertime pls